integration_tests/utils.py

```diff
@@ -12,7 +12,7 @@
 from pystarport.ports import rpc_port
 
 
-def wait_for_block(cli, height, timeout=60):
+def wait_for_block(cli, height, timeout=6000):
```

Comment:
is the timeout in seconds? if so, 100 minutes may be too high?

Reply:
yes, changed to 120seconds now, should be enough.